windows/RNGetRandomValues.cpp

```diff
@@ -0,0 +1,11 @@
+#include "pch.h"
+#include "RNGetRandomValues.h"
+
+namespace winrt::RNGetRandomValues::implementation
+{
+    hstring RNGetRandomValues::getRandomBase64(int byteLength)
```

Comment:
consider returning std::string (via conversion to utf8) instead of hstring to not leak WinRT-isms